main.cpp

```diff
@@ -48,50 +50,53 @@ void main_window::on_paint(HDC hdc) {
 	}
 	else {
 		for (int i = 0; i < default_number; ++i) {
-			double a = 2 * M_PI * i / default_number;
+			double a = 2 * std::numbers::pi * i / default_number;
 			double x = rc.right / 2 + r * cos(a);
 			double y = rc.bottom / 2 + r * sin(a);
 			::Ellipse(hdc, x + r, y + r, x - r, y - r);
 		}
 	}
-
+	SelectObject(hdc, brush);

```

Comment:
Kako će selektiranje istog objekta još jednom pomoći da on ne bude selektiran?
Morate selektirati prethodni objekt (ono što je **vratio** prethodni poziv `SelectObject`, pogledajte predavanja i primjere).